Build failure for armv7-unknown-linux-uclibceabihf

```sh
$ cargo +nightly --version
cargo 1.96.0-nightly (888f67534 2026-03-30)
$ cargo +nightly check -Zbuild-std --target armv7-unknown-linux-uclibceabihf --all-targets --all-features
```

```log
error[E0425]: cannot find value `AT_MINSIGSTKSZ` in crate `libc`
   --> /home/runner/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/std/src/sys/pal/unix/stack_overflow.rs:310:63
    |
310 |         let dynamic_sigstksz = unsafe { libc::getauxval(libc::AT_MINSIGSTKSZ) };
    |                                                               ^^^^^^^^^^^^^^
    |
   ::: /home/runner/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/libc-0.2.183/src/unix/linux_like/linux/uclibc/mod.rs:400:1
    |
400 | pub const MINSIGSTKSZ: c_int = 2048;
    | ---------------------------- similarly named constant `MINSIGSTKSZ` defined here
    |
help: a constant with a similar name exists
    |
310 -         let dynamic_sigstksz = unsafe { libc::getauxval(libc::AT_MINSIGSTKSZ) };
310 +         let dynamic_sigstksz = unsafe { libc::getauxval(libc::MINSIGSTKSZ) };
    |

For more information about this error, try `rustc --explain E0425`.
error: could not compile `std` (lib) due to 1 previous error
Error: Process completed with exit code 101.
```

cc: @skrap

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -360,7 +360,9 @@ jobs:
           - target: x86_64-unknown-dragonfly
           - target: x86_64-unknown-openbsd
           - target: x86_64-unknown-haiku
-          - target: armv7-unknown-linux-uclibceabihf
+          # Temporarily disable armv7-unknown-linux-uclibceabihf
+          # https://github.com/rust-lang/rust/issues/154679
+          # - target: armv7-unknown-linux-uclibceabihf
           # Disable Hurd due to 
           #     1. https://github.com/rust-lang/libc/issues/4421
           #     2. https://github.com/nix-rust/nix/pull/2635#issuecomment-2842062528

